client/src/App.js

```diff
@@ -3,13 +3,29 @@ import GlobalStyles from "./GlobalStyles";
 import Home from "./pages/Home";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import Paste from "./pages/Paste";
+import Confirmation from "./components/Confirmation";
 
 function App() {
+  const [showConfirmation, setShowConfirmation] = React.useState(false);
+
+  function isPasteCreator() {
+    setShowConfirmation(true);
+  }
+
   return (
     <div className="App">
       <GlobalStyles />
+      {showConfirmation && (
+        <Confirmation>
+          <div>🎉🎉🎉</div>URL copied to your clipboard!
+        </Confirmation>
+      )}
       <Router>
-        <Route exact path="/" component={Home} />
+        <Route
+          exact
+          path="/"
+          component={() => <Home isPasteCreator={isPasteCreator} />}
```

Comment:
`isPasteCreator` is misleading because I would suggest having a boolean instead of a function

Reply:
`onPaste` maybe